applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -248,16 +357,42 @@ static void stream_configured_cb(struct bt_audio_stream *stream,
 		return;
 	}
 
+	new_pd_us = find_headset_pres_delay(CONFIG_BT_AUDIO_PRES_DLY_SRCH_MODE, channel_index);
+	if (new_pd_us == 0) {
+		LOG_ERR("Cannot get a valid presentation delay");
+	}
+
+	LOG_DBG("New presentation delay: %d us", new_pd_us);
+	lc3_preset_sink.qos.pd = new_pd_us;
```

Comment:
To me it is not optimal that we use the preset to store theis new_pd_us value. To me a preset should stay constant

Reply:
This has been removed in the new code.